_source/logzio_collections/_metrics-sources/cassandra.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Ship Cassandra metrics
+logo:
+  logofile: cassandra_logo.svg
+  orientation: vertical
+data-source: Cassandra
+contributors:
+  - daniel-tk
+  - shalper
+shipping-tags:
+  - database
+---
+
+You can ship Cassandra metrics to Logz.io using Metricbeat.
+
+
+#### Configuration
+
+**Before you begin, you'll need**:
+
+* A server or cluster with Cassandra installed
+* [Metricbeat 7.1](https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-installation.html) or higher installed
+
+<div class="tasklist">
+
+### Expose metrics
+
+The first step is to expose Cassandra metrics using the [Prometheus JMX exporter](https://github.com/prometheus/jmx_exporter).
+
+
+
+
+##### JMX exporter
+
+In your Cassandra server, go to the directory where Cassandra is installed on your machine:
+
+```shell
+cd {path_to_cassandra_dir}
+```
+
+Download the Prometheus JMX exporter:
+
+```
+curl -L -O https://search.maven.org/remotecontent?filepath=io/prometheus/jmx/jmx_prometheus_javaagent/0.13.0/jmx_prometheus_javaagent-0.13.0.jar
+```
+
+Download the exporter configuration file for Cassandra:
+
+```
+curl -L -O https://raw.githubusercontent.com/prometheus/jmx_exporter/master/example_configs/cassandra.yml
+```
+
+##### Configure Cassandra to forward metrics to Metricbeat
+
+You will need to repeat the following procedure for each Cassandra node that will be sending metrics to Logz.io.
```

Comment:
I guess it wasn't clear on my side, but this sentence belong to step no.1 (JMX exporter), not to step no.2
Meaning they have to download the jmx exporter in each casandra node